crates/diesel_models/src/query/customers.rs

```diff
@@ -139,38 +144,28 @@ impl Customer {
         .await
     }
 
+    #[cfg(all(feature = "v2", feature = "customer_v2"))]
     pub async fn find_by_customer_id_merchant_id(
```

Comment:
this function is not required

Reply:
This is required since the other function is using generic_find_by_id which expects composite key where as in v2 we donot have composite primary key concept